src/cli/cli.cc

```diff
@@ -70,6 +70,66 @@ static const std::unordered_map<InstallResult::Status, StatusCode> i2s = {
     {InstallResult::Status::UnknownError, StatusCode::UnknownError},
 };
 
+using SC = StatusCode;
```

Comment:
I would rather use this alias for the overall file (define on top and apply to all cli.cpp) or drop it and just use `StatusCode` as it is defined everywhere.

Reply:
Done.